AUBGTheHUB/monolith
# #507 Fix bug with error messages

## Description
Conditional case for setting appropriate error message was failing due to accessing a non-existent index. 
Extracted message handling because the same code is being used at two places.

## How to verify
Try to run through all erroring conditions and see if messages get displayed.

Resolves #507
